packages/unplugin-typegpu/README.md

```diff
@@ -0,0 +1,50 @@
+<div align="center">
+
+# unplugin-typegpu
+
+🚧 **Under Construction** 🚧 - [GitHub](https://github.com/software-mansion/TypeGPU/tree/main/packages/unplugin-typegpu)
+
+</div>
+
+Build plugins for [TypeGPU](https://typegpu.com), transpiling JavaScript functions into WGSL at build-time.
```

Comment:
Could be misinformation 😂. Maybe we can rephrase it while not going into more detail than necessary.

```suggestion
Build plugins for [TypeGPU](https://typegpu.com) that enable:
- Seamless JavaScript -> WGSL transpilation
- [🚧 TODO] Improved debugging with automatic naming of resources
```